

kernel/sched.c:4114: warning: `arch_destroy_sched_domains' defined but not used

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/kernel/sched.c |    2 ++
 1 files changed, 2 insertions(+)

diff -puN kernel/sched.c~sched-arch_destroy_sched_domains-warning-fix kernel/sched.c
--- 25/kernel/sched.c~sched-arch_destroy_sched_domains-warning-fix	2004-10-05 02:09:47.618591208 -0700
+++ 25-akpm/kernel/sched.c	2004-10-05 02:09:47.624590296 -0700
@@ -4570,10 +4570,12 @@ static void __devinit arch_init_sched_do
 	}
 }
 
+#ifdef CONFIG_HOTPLUG_CPU
 static void __devinit arch_destroy_sched_domains(void)
 {
 	/* Do nothing: everything is statically allocated. */
 }
+#endif
 
 #undef SCHED_DOMAIN_DEBUG
 #ifdef SCHED_DOMAIN_DEBUG
_
